src/db/postgres/lrsql/postgres/record.clj

```diff
@@ -66,7 +66,9 @@
       (migrate-agent-profile-document-last-modified! tx pd/local-tz-input))
     (if (-> tuning :config :enable-jsonb)
       (migrate-to-jsonb! tx)
-      (migrate-to-json! tx)))
+      (migrate-to-json! tx))
+    (when (= [] (check-statement-to-actor-cascading-delete tx))
```

Comment:
this is bordering on nitpicky but in general is a good thing to know. If you annotate the SQL query with 
`-- :result :one`
it will return nil on no matches, which will make this a little cleaner than an empty vec comparison, i.e.
`(some? (check-statement-to-actor-cascading-delete tx))`

Reply:
i am a nil supremacist, will change